README.md

```diff
@@ -1 +1,32 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> A responsive leaderboard webpage made with JavaScript and Webpack, made for a Microverse project
+
+## Built With
+
+- HTML, CSS, SASS, JavaScript, Webpack
+
+## Author
+
+👤 **Maximiliano Ungredda**
+
+- GitHub: [@renercrows](https://github.com/renercrows)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ **OPTIONAL** ] I suggest you add the **Getting Started** and **Setup** sections in your README with information about cloning the repo, accessing the cloned directory, how to install all dependencies, how to run build for production mode, and the **most important how to run the server**. You can use this **[template](https://github.com/tresorsawasawa/Leaderboard_App/blob/development/README.md)** to help you complete all the necessary information.